scrub_hub_vite/src/models/chat.ts

```diff
@@ -0,0 +1,197 @@
+import { fromBase64, toBase64 } from "../base64";
+import { ChatProps, MessageProps } from "../components/ChatComponent";
+import { ConversationListProps, ConversationProps } from "../components/ConversationListComponent";
+
+export type User = { name: string, id: number };
+function isUser(obj?: {[key: string]: unknown}): obj is User {
+	return !!(
```

Comment:
What tf is going on here LOL It looks like you are converting the result to a boolean but this method is TERRIBLE practice haha

Reply:
If you are converting something to boolean use Boolean(expression) instead of !!expression :) Just more readable, and does the same thing